test/Clownface/literal.js

```diff
@@ -0,0 +1,111 @@
+/* global before, describe, it */
+
+const assert = require('assert')
+const clownface = require('../..')
+const rdf = require('rdf-ext')
+const rdfDataModel = require('@rdfjs/data-model')
+const initExample = require('../support/example')
+const Clownface = require('../../lib/Clownface')
+
+describe('.literal', () => {
+  let graph
+
+  before(() => {
+    return initExample().then(dataset => {
+      graph = dataset
+    })
+  })
+
+  it('should be a function', () => {
+    const cf = clownface(graph)
+
+    assert.strictEqual(typeof cf.literal, 'function')
+  })
+
+  it('should return a new Clownface instance', () => {
+    const cf = clownface(graph)
+
+    const result = cf.literal('example')
+
+    assert(result instanceof Clownface)
+    assert.notStrictEqual(result, cf)
+  })
+
+  it('should use the dataset from the context', () => {
+    const cf = clownface(graph)
+
+    const result = cf.literal('example')
+
+    assert.strictEqual(result._context[0].dataset, graph)
+  })
+
+  it('should use the given string as Literal', () => {
+    const value = '2311 North Los Robles Avenue, Aparment 4A'
+    const cf = clownface(graph)
+
+    const result = cf.literal(value)
+
+    assert.strictEqual(result._context.length, 1)
+    assert.strictEqual(result._context[0].term.termType, 'Literal')
+    assert.strictEqual(result._context[0].term.value, value)
+  })
+
+  it('should use the given number as Literal', () => {
+    const value = 123
+    const cf = clownface(graph)
+
+    const result = cf.literal(value)
+
+    assert.strictEqual(result._context.length, 1)
+    assert.strictEqual(result._context[0].term.termType, 'Literal')
+    assert.strictEqual(result._context[0].term.value, value.toString())
```

Comment:
Should also expect a numeric datatype here?

Reply:
Created #10 for it. Let's have a look at it in another PR.